pycyphal/dsdl/__init__.py

```diff
@@ -21,7 +21,8 @@
 from ._compiler import compile_all as compile_all
 from ._compiler import GeneratedPackageInfo as GeneratedPackageInfo
 
-from ._import_hook import install_import_hook as install_import_hook
+from ._import_hook import add_import_hook as add_import_hook
```

Comment:
This change breaks the API. The ticket calls for deprecation of the old name, not its removal. Please add an `install_import_hook` wrapper here in this file and make it deprecated, similar to `generate_package`.